python/semantic_kernel/utils/settings.py

```diff
@@ -231,7 +235,7 @@ def redis_settings_from_dot_env() -> str:
     return connection_string
 
 
-def azure_aisearch_settings_from_dot_env() -> Tuple[str, str]:
+def azure_aisearch_settings_from_dot_env() -> Tuple[str, str, str]:
```

Comment:
I would prefer the index to be driven by a include_index_name variable, and then returning 2 or 3 items in the Tuple, so that this is not a breaking change.

Reply:
Good call. Updated this.